### PR TITLE
Add Atlassian Confluence

### PR DESCRIPTION
**Pull request by int128**
**What this PR does / why we need it**:

This PR adds a new chart for Atlassian Confluence.
Atlassian Confluence is an enterprise wiki collaboration platform.

**Special notes for your reviewer**:

This chart uses the official image [atlassian/confluence-server](https://bitbucket.org/atlassian/docker-atlassian-confluence-server).

I am running a Confluence server with this chart in my environment. See also https://github.com/int128/devops-kompose.

### CODE REVIEW

File: stable/atlassian-confluence/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```
Comment (mattjmcnaughton) [inline stable/atlassian-confluence/Chart.yaml:1]: Could you please specify the owners of the chart in this file?

**Review by mattjmcnaughton — COMMENTED** [review]
Hi @int128 - thanks for your work on this pr!

For this chart to immediately enter the `stable` repository, it needs to meet the following [technical requirements](https://github.com/kubernetes/charts/blob/master/CONTRIBUTING.md#technical-requirements).

It looks to me like it met most (if not all) of them - great work!

The only ones I didn't have enough experience to evaluate on are:
- Support application upgrades
- Allow customization of the application configuration
- Provide a secure default configuration

Do you mind commenting on those?

File: stable/atlassian-confluence/Chart.yaml (hunk 1 above)
Comment (int128) [reply stable/atlassian-confluence/Chart.yaml:1]: Sorry, this is my first pr :-) and tell me whether I need to put my name `int128` or your name `mattjmcnaughton` in the owners.

**Comment by int128** [comment]
Thank you for the review!

> Support application upgrades

Just bump a chart and Confluence automatically upgrades database on boot up.
I added description to README.md.

> Allow customization of the application configuration

Confluence does not provide config files or env-vars.
We need to configure database and etc on the setup wizard on a web browser.

> Provide a secure default configuration

Currently this chart does not use any credentials such as database password.
Is more configuration needed?

File: stable/atlassian-confluence/Chart.yaml (hunk 1 above)
Comment (int128) [reply stable/atlassian-confluence/Chart.yaml:1]: Got it. I added `maintainers`.

**Comment by int128** [comment]
Any update on this? Please let me know any suggestions.

**Comment by Hokwang** [comment]
I am looking forward to merging this!

**Comment by Hokwang** [comment]
@int128 how about reopen this ?
I think you can manage charts for confluence and jira in this official helm repo not your personal repo.
And It would be help for other users.

**Pull request closed without merging**